app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -8,33 +8,52 @@
 /** Holds all custom binding adapters that set margin values. */
 public final class MarginBindingAdapters {
 
-  /** Used to set a margin-start for views. */
+  public static boolean isRtlLayout(View view) {
+    return view.getLayoutDirection() == View.LAYOUT_DIRECTION_RTL;
+  }
+
+  /** Sets the start margin for a view, accounting for RTL scenarios. */
   @BindingAdapter("app:layoutMarginStart")
   public static void setLayoutMarginStart(@NonNull View view, float marginStart) {
     if (view.getLayoutParams() instanceof MarginLayoutParams) {
       MarginLayoutParams params = (MarginLayoutParams) view.getLayoutParams();
-      params.setMargins(
-          (int) marginStart,
-          params.topMargin,
-          params.getMarginEnd(),
-          params.bottomMargin
-      );
+      float marginEnd = params.getMarginEnd();
+      setLayoutDirectionalMargins(view, (int) marginEnd, params, (int) marginStart);
       view.requestLayout();
     }
   }
 
-  /** Used to set a margin-end for views. */
+  /** Sets the end margin for a view, accounting for RTL scenarios. */
   @BindingAdapter("app:layoutMarginEnd")
   public static void setLayoutMarginEnd(@NonNull View view, float marginEnd) {
     if (view.getLayoutParams() instanceof MarginLayoutParams) {
       MarginLayoutParams params = (MarginLayoutParams) view.getLayoutParams();
+      float marginStart = params.getMarginStart();
+      setLayoutDirectionalMargins(view, (int) marginEnd, params, (int) marginStart);
+      view.requestLayout();
+    }
+  }
+
+  private static void setLayoutDirectionalMargins(
```

Comment:
Some nits:
1. Indentation for these should be 2 spaces
2. We can we-retrieve the params rather than needing to pass them in again (since we have the view, anyway)
3. marginStart should be before marginEnd (as is conventional in Android)

Reply:
Done.

Reply:
Point (1) still seems unaddressed. Ditto elsewhere in PR (we should use +2 indentation for Java in these cases).

Reply:
Done.

Reply:
This still seems unresolved. This is how the indentation should be:

```kotlin
  private static void setLayoutDirectionalMargins(
    @NonNull View view,
    int marginStart,
    int marginEnd
  ) {
    MarginLayoutParams params = (MarginLayoutParams) view.getLayoutParams();
    params.setMargins(
      marginStart,
      params.topMargin,
      marginEnd,
      params.bottomMargin
    );
  }
```

Ditto elsewhere.

Also: please don't mention something is 'done' without the changes being applied. If you're unsure what's being asked, please ask for clarification.

Reply:
This is already fixed.

Reply:
No, it isn't. This is the current version on GitHub: https://github.com/oppia/oppia-android/blob/2c454f307dd23a650bbff9071488e1b194e99ecc/app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java#L44

It has 4 spaces of indentation, not 2. If this is fixed for you locally, are you pushing it?

Reply:
It appears fixed now. Thanks!